<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CavityOutput</class>
 <widget class="QDialog" name="CavityOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cavity Output</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>MultiCell Output Parameters</string>
   </property>
   <widget class="QLabel" name="l_minBeta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Min beta in range:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_maxBeta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Max beta in range:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_FieldPeaks">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Field Peaks:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Replot">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Replot</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_peaks">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l_freq">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>244</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l_RQ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>244</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l_QBCS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>244</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l_Epkk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>244</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l_Hpkk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>244</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_phii">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Synchronous Phase:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tE_minBeta">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tE_maxBeta">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tE_phii">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>531</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Transit Time Factor Calculation</string>
   </property>
   <widget class="QPushButton" name="pb_TTF">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Numerical TTF Curve and fit effective beta</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_TTF">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the beta range to evaluate TTF</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>441</width>
     <height>561</height>
    </rect>
   </property>
   <property name="title">
    <string>Field profile and Cavity Geometry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Print">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>580</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Quit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>580</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>pb_Print</zorder>
  <zorder>pb_Quit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
